source/crud/read-operations/skip.txt

```diff
@@ -0,0 +1,58 @@
+=====================
+Skip Returned Results
+=====================
+
+.. default-domain:: mongodb
+
+
+Use skip to omit documents from the beginning of the list of returned
+documents for a read operation. You can combine skip with sort to omit
+the top (for descending order) or bottom (for ascending order) ``N``
+results for a given query. Since the order of documents returned is
+:manual:`not guaranteed
```

Comment:
I wonder if this link would work better around "order of documents" as I would otherwise expect to be linked to a document section called "not guaranteed" which, to me, is both confusing without context and has some negative connotation associated.

Reply:
Done.

Reply:
Similar comment for the `read-operations/sort.txt` file.